src/export/digitalPlanning/model.ts

```diff
@@ -716,42 +716,38 @@ export class DigitalPlanning {
   private getFiles(): Payload["files"] {
     const files: File[] = [];
 
-    Object.entries(this.passport.data)
-      .filter(([, v]) => v?.[0]?.url)
-      .forEach(([key, arr]) => {
-        (arr as { url: string }[]).forEach(({ url }) => {
-          try {
-            // push a new label to an existing file
-            if (files.filter((file) => file.name === url).length > 0) {
-              files
-                .filter((file) => file.name === url)
-                .map((file) => {
-                  file["type"].push({
-                    value: key,
-                    description: this.findDescriptionFromValue("FileType", key),
-                  } as FileType);
-                });
-            } else {
-              // add a new file
-              files.push({
-                name: url,
-                type: [
-                  {
-                    value: key,
-                    description: this.findDescriptionFromValue("FileType", key),
-                  } as FileType,
-                ],
-                description: extractFileDescriptionForPassportKey(
-                  this.passport.data,
-                  key,
-                ),
-              });
-            }
-          } catch (err) {
-            throw new Error(`Error formatting files: ${err}`);
-          }
-        });
-      });
+    this.passport.files.forEach(({ url, key }) => {
```

Comment:
one small question: can we confirm that we'll still pick up location plan files if we read from `files` rather than `data`? I don't see a test case covering that? It's a bit of an edge case as it's the only way to upload a file outside of an upload component.

Reply:
Very good shout - will add a test case 👍